static_sources/eateryAttributes.json

```diff
@@ -0,0 +1,67 @@
+{
+  "104-West": {
+    "reserveLink": "https://www.opentable.com/restaurant/profile/1080703"
```

Comment:
by convention we usually would do something like `reserveUrl`

Reply:
yeah that's my bad I just used the first non OpenTable variable name I thought of